File: COJ/4079.cpp
#include<bits/stdc++.h>
using namespace std;

int main(){
  float R, S;
  int N;
  cin >> R >> S;
  cin >> N;

  float anw = 0;
  for(int i = 0; i < N; i++){
    float F;
    cin >> F;
    float diff = ((1.0*(R-F))*1.0) / (S*1.0);
    anw+=diff;
  }
  anw+= 2*N;
  // cout << setprecision(6);
  cout << anw << endl;
  return 0;
}

File: COJ/4079.py
import sys

R,S = map(float, sys.stdin.readline().split())
N = int(sys.stdin.readline())
anw = 0
for i in range(0, N):
  F = float(sys.stdin.readline())
  diff = (R-F) / S
  anw+= diff

anw+= 2*N
print("{0:.4f}".format(anw))
